azure-core/azure/core/pipeline/policies/retry.py

```diff
@@ -115,7 +114,8 @@ def get_backoff_time(self, settings):
         backoff_value = settings['backoff'] * (2 ** (consecutive_errors_len - 1))
         return min(settings['max_backoff'], backoff_value)
 
-    def parse_retry_after(self, retry_after):
+    @staticmethod
+    def parse_retry_after(retry_after):
```

Comment:
Let's not make this static. I suspect that we may end up subclassing this type to support things like `x-ms-retry-after` headers (where the value is in milliseconds)